src/transformers/training_args.py

```diff
@@ -1618,7 +1618,7 @@ def __post_init__(self):
         if (
             self.framework == "pt"
             and is_torch_available()
-            and (self.device.type == "cpu" and (version.parse(get_torch_version()) < version.parse("2.3.0")))
+            and (self.device.type == "cpu" and is_torch_greater_or_equal_than_2_3)
```

Comment:
I think? 

```suggestion
            and (self.device.type == "cpu" and not is_torch_greater_or_equal_than_2_3)
```

Reply:
Yep! (sorry had computer in front of me and pushed right as you caught it 😓 )